src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -219,6 +219,15 @@
   <data name="AbsolutePathExpected" xml:space="preserve">
     <value>Absolute path expected.</value>
   </data>
+  <data name="EmptyKeyInPathMap" xml:space="preserve">
+    <value>A key in the pathMap is empty.</value>
+  </data>
+  <data name="EmptyValueInPathMap" xml:space="preserve">
+    <value>A value in the pathMap is empty.</value>
```

Comment:
Not with the current spec, as we always insert a path separator. But I can imagine changing it.